### DISCUSSION TITLE
Предложения по графике:

### DISCUSSION THREAD

**Discussion by 1Pixel1** [OGSR/OGSR-Engine#282] [Ideas]
1. Если в шейдере accum_emissive.ps поменять код на следующий, то это пофиксит пересвеченные глаза монстров, лампочки итд (подсмотрено в ECB shaderpack-rev.0014-rc6):

#include "common.h"
half4  main  ( p_flat I )  : COLOR
{
  return   (half4) 1.1f;
}

2. Grass waves https://github.com/dsh2dsh/op2ogse/commit/a44886507b2b6beb06d4caae00c93a2eface9b0d

3. Намокание на DX9.

**Comment by xrSimpAI** [comment]
1. Сильно не уверен, что это нужно.
2. Это отсюда и взято как бы.
3. Нет смысла делать граф фичи на дх9, т.к. дх9 тут оставлен по сути для владельцев бомж-пека, или для тек у кого глючит дх11. Это очень маленький % юзеров. Я бы вообще выпилил дх9, если б в огср моде дх11 не глючил иногда у некоторых.
